requirements/edx/pre.txt

```diff
@@ -4,15 +4,14 @@
 #   * @edx/ospr - to check licensing
 #   * @edx/devops - to check system requirements
 
-# Use a modern setuptools instead of distribute
-setuptools==18.0.1
+# Packaging pre-requisites.
+pyparsing==2.0.7
+setuptools==34.0.2
```

Comment:
I put pyparsing here because we had it in our base requirements.  I think because of using these requirements incrementally (on top of an older install of older requirements), we'd have pyparsing 2.0.1 in place, and setuptools would fail.  This forces pyparsing to a good version in time for setuptools to succeed.

I don't quite understand how setuptools expects to get its requirements installed.  This configuration works.  We could choose an "abundance of caution" approach and pin everything that setuptools uses, but the other philosophy is to let the environment provide it somehow.